content/en/agent/amazon_ecs/_index.md

```diff
@@ -0,0 +1,315 @@
+
+## Overview
+
+Amazon ECS on EC2 is a highly scalable, high-performance container management service for Docker containers running on EC2 instances.
+
+This page covers Amazon ECS setup with [Datadog Container Agent v6][1]. For other setups, see:
+
+- [Datadog Container Agent v5 setup for Amazon ECS][2]
+- [Datadog Host Agent setup with Autodiscovery][3]
+
+## Setup
+
+To monitor your ECS containers and tasks with Datadog, run the Agent as a container on every EC2 instance in your ECS cluster. As detailed below, there are a few setup steps:
+
+1. Add an ECS Task
+2. Create or modify your IAM Policy
+3. Schedule the Datadog Agent as a Daemon Service
+
+If you don't have a working EC2 Container Service cluster configured, review the [Getting Started section in the ECS documentation][4].
+
+### Metric collection
+
+#### Create an ECS Task
+
+This task launches the Datadog container. When you need to modify the configuration, update this task definition as described [further down in this guide][#create-or-modify-your-iam-policy].
+
+If you're [using APM][5], [DogStatsD][6], or [log management][7], set the appropriate flags in the task definition:
+
+- If you are using APM, set `portMappings` so your downstream containers can ship traces to the Agent service. APM uses `TCP` on port `8126` to receive traces, so set this as the `hostPort` in your task's definition. 
+
+**Note**: To enable trace collection from other containers, ensure that the `DD_APM_NON_LOCAL_TRAFFIC` environment variable is set to `true`. Learn more about [APM with containers][8].
+
+- If you are using DogStatsD, set the `hostPort` as `UDP` on port `8125` in your task's definition. 
+
+**Note**: To enable DogStatsD metrics collection from other containers, ensure the `DD_DOGSTATSD_NON_LOCAL_TRAFFIC` environment variable is set to `true`.
+
+- If you are using log management, refer to the dedicated [Log collection documentation][5].
+
+Double check the security group settings on your EC2 instances. Make sure these ports are not open to the public. Datadog uses the private IP to route to the Agent from the containers.
+
+Configure the task using either the [AWS CLI tools][9] or using the Amazon Web Console.
+
+{{< tabs >}}
+{{% tab "AWS CLI" %}}
+
+1. For Linux containers, download [datadog-agent-ecs.json][1] ([datadog-agent-ecs1.json][2] if you are using an original Amazon Linux AMI). For Windows, download [datadog-agent-ecs-win.json][3].
+2. Edit `datadog-agent-ecs.json` and set `<YOUR_DATADOG_API_KEY>` with the [Datadog API key][4] for your account.
+3. Optionally - Add an [Agent health check](#agent-health-check).
+4. Optionally - If you are in Datadog EU site, edit `datadog-agent-ecs.json` and set `DD_SITE` to `DD_SITE:datadoghq.eu`.
+5. Optionally - See [log collection][18] to activate log collection.
+6. Optionally - See [process collection](#process-collection) to activate process collection.
+7. Optionally - See [trace collection (APM)][16] to activate trace collection.
```

Comment:
same as above. We can strip this from here to keep it shorter and simpler to get started

Reply:
Sure, let's leave as is for now.